src/Utility/String.php

```diff
@@ -1,5 +1,5 @@
 <?php
 // @deprecated Backward compatibility with 2.x series
-if (PHP_VERSION_ID < 70000) {
```

Comment:
Why? If you got < 7.0 you get no error.
If you use >= 7.0.0 you will either get an error if you do not alias the class (use Cake\Utility\String as CakeString) - or - if you alias the class - you will now get the notice... anyway I will re-add it. Just not sure it is required.